http/install-chroot.sh

```diff
@@ -66,13 +66,31 @@ ExecStart=/usr/bin/pacman-key --populate archlinux
 WantedBy=multi-user.target
 EOF
 
+# Add service for running reflector on first boot
+cat <<EOF >/etc/systemd/system/reflector-init.service
+[Unit]
+Description=Initializes mirrors for the VM
+After=network.target
+Wants=network.target
+ConditionFirstBoot=yes
```

Comment:
Pretty nice use of `ConditionFirstBoot` :)

Reply:
I am just not sure about that reflector command.. normally you want to specify the country there, but we can't know where our VM will live.. So Actually we would need a tool that measures the distance from the client to all mirrors.. and then chooses the best one.